src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -26281,11 +26281,9 @@ class Attr3 : System.Attribute {}
             Assert.Equal(1, analyzer.FireCount7);
             Assert.Equal(1, analyzer.FireCount8);
             Assert.Equal(1, analyzer.FireCount9);
-            Assert.Equal(1, analyzer.FireCount10);
```

Comment:
@AlekseyTs I had to adjust the existing tests to match the new behavior. This leads to holes now in the `FireCountXXX` counter in the analyzer. Please let me know if current change to tests looks fine or you'd prefer I adjust the `XXX` suffixes to avoid the counter holes.

Reply:
>  I had to adjust the existing tests to match the new behavior. This leads to holes now in the FireCountXXX counter in the analyzer. Please let me know if current change to tests looks fine or you'd prefer I adjust the XXX suffixes to avoid the counter holes.

Are you saying you remove some of the fields because they become obsolete? I am fine with the gaps in names.

Reply:
Yes, that is correct